docs/docusaurus/docs/cloud/connect/connect_databrickssql.md

```diff
@@ -13,30 +13,20 @@ import Tabs from '@theme/Tabs';
 
 - You have a Databricks SQL catalog, schema, and table.
 
-- To improve data security, GX recommends creating a separate Databricks SQL [service principal](https://docs.databricks.com/en/admin/users-groups/service-principals.html#manage-service-principals-in-your-account) for your GX Cloud connection.
+- Optional. To improve data security, GX recommends creating a separate Databricks SQL [service principal](https://docs.databricks.com/en/admin/users-groups/service-principals.html#manage-service-principals-in-your-account) for your GX Cloud connection.
 
+## Connect to a Databricks SQL Data Source and add a Data Asset
 
-## Connect to a Databricks SQL Data Asset
-
-1. In GX Cloud, click **Data Assets** > **New Data Asset** > **Databricks SQL**.
+1. In GX Cloud, click **Data Assets** > **New Data Asset** > **New Data Source** > **Databricks SQL**.
 
 2. Enter a meaningful name for the Data Source in the **Data Source name** field.
 
-3. Enter a connection string in the **Connection string** field. The connection string format is `databricks://token:{token}@{host}?http_path={http_path}&catalog={catalog}&schema={schema}`.
-    - Check the instructions to create a GX-specific user in your Databricks SQL catalog by clicking "See instructions"
+3. Enter a connection string in the **Connection string** field. The connection string format is `databricks://token:{token}@{host}:{port}?http_path={http_path}&catalog={catalog}&schema={schema}`. If you created a separate Databricks SQL service principal for your GX Cloud connection as recommended above, use those credentials in the connection string.
```

Comment:
I tried to add a suggested change here, but the git diff preview was not cooperating - what do you think about moving all the connection string formats (in Postgres, Databricks, and Snowflake) to a codebox so that the strings don't break weirdly in the text?

For example:

---

3. Enter a connection string in the **Connection string** field. The connection string format is:
   ```
   databricks://token:{token}@{host}:{port}?http_path={http_path}&catalog={catalog}&schema={schema}
   ```
   If you created a separate Databricks SQL service principal for your GX Cloud connection as recommended above, use those credentials in the connection string.

Reply:
I like this idea!

I wasn't sure what to use as the language identifier for syntax highlighting. I went with python for now but please let me know if something else would be more appropriate.

For snowflake and databricks, the connection string is still automatically wrapping in the code block. But for postgres I see a button to toggle the wrap on or off. I'm not sure why these are behaving differently but still think it's an improvement over the inline wrapping. 
<img width="774" alt="Screenshot 2024-11-22 at 2 30 45 PM" src="https://github.com/user-attachments/assets/dfc6eeed-f255-47e2-82ea-e966ce810ad2">

Reply:
Python as the syntax highlighting makes sense to me!